lua/entities/lunasflightschool_missile.lua

```diff
@@ -246,10 +253,12 @@ else
 		self.snd = CreateSound( self, "weapons/flaregun/burn.wav" )
 		self.snd:Play()
 
-		local effectdata = EffectData()
-			effectdata:SetOrigin( self:GetPos() )
-			effectdata:SetEntity( self )
-		util.Effect( "lfs_missile_trail", effectdata )
+		if CLIENT then
+			local effectdata = EffectData()
+				effectdata:SetOrigin( self:GetPos() )
+				effectdata:SetEntity( self )
+			util.Effect( "lfs_missile_trail", effectdata )
+		end

```

Comment:
You can still run this serverside if you want, you just need to provide `true` to the fourth argument

```suggestion
		local effectdata = EffectData()
			effectdata:SetOrigin( self:GetPos() )
			effectdata:SetEntity( self )
		util.Effect( "lfs_missile_trail", effectdata, true, true )

```

[ref](https://wiki.facepunch.com/gmod/util.Effect)

Reply:
Applied.